charts/zigbee2mqtt/values.yaml

```diff
@@ -0,0 +1,103 @@
+# Default values for zigbee2mqtt.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: koenkk/zigbee2mqtt
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+timezone: "Europe/Amsterdam"
```

Comment:
Would you consider defaulting this to `UTC` ?

Reply:
Sure